app/gradle.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
The file should be removed, right now its committed as an empty file

Reply:
I can test if it works without the file being present, but I am not going to remove it if it doesn't.

Reply:
gradle semantic release generates the file if it doesn't exist, which is correct behaviour because the props file is not something a gradle project depends on